markdown/developers/tutorials/adding_modules_to_pipelines.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Tutorial: Adding Modules to Pipelines'
+subtitle: Quick reference of the steps needed to add a new module to a pipeline.
+---
+
+This tutorial is a quick reference overview of the steps you will normally take when adding a new module to a pipeline.
+
+Different pipelines may have different workflows, but the following steps will cover the most common aspects.
+
+1. Install module to the pipeline
+
+   ```bash
+   nf-core modules install <tool>/<subtool>
+   ```
+
+2. Add `include` statement at the top of the pipeline's (sub)workflow to import module
+3. Insert module execution into the relevant place (sub)workflow script `TOOL(ch_input)`
+4. Create a section in `conf/modules.conf` for the module (with a default results directory and output file pattern)
+5. Add any necessary parameters for the module with defaults to `nextflow.config`
+6. Insert the parameters into `ext.args` of corresponding `conf/modules.config` (and in some cases in the module reference in the workflow script)
+7. Update the `nextflow_schema.json` to include new parameters with
+
+   ```bash
+   nf-core schema build
+   ```
+
+8. Update `assets/multiqc_config.yml` to include any new MultiQC modules (if any exist) and specify order in the report
+9. Add a citation for the new tool/module to `citations.md`
+10. Update `docs/USAGE.md` to describe any important information about running of the module (this can be optional in some cases)
+11. Update `docs/OUTPUT.md` to describe the directories output files of the module
+12. Update `README.md` mentioning the tool is used (optional)
+13. Run prettier formatting on the whole repository (if installed)
+
+    ```bash
+    prettier -w .
+    ```
+
+    - Or run VSCode 'Format Document' function on: `citations.md`, `usage.md`, `output.md`, `README.md`, `modules.json`, `nextflow_schema.json`
+    - If you forget this step you can also post a comment on the open PR once made with `@nf-core bot fix linting`
+
```

Comment:
should it not be `@nf-core-bot fix linting` with a dash?

Reply:
Judging from the contents of the `.github/workflows` it might be correct https://github.com/nf-core/nf-co.re/blob/a0630dafeffa7262aad2616266478405265e953f/.github/workflows/fix-linting.yml#L11  🤔

Reply:
Correct!